src/sentry/static/sentry/app/views/settings/components/forms/controls/radioGroup.jsx

```diff
@@ -19,13 +19,14 @@ const RadioGroup = ({value, disabled, choices, label, onChange, ...props}) => {
           index={index}
           tabIndex={isSelected(id) ? 0 : -1}
           aria-checked={isSelected(id)}
+          disabled={disabled}
```

Comment:
I think this still has the same problem of generating that DOM warning.

Reply:
I rejigged the properties based on @billyvg's suggestion. There aren't any DOM warnings now.